pkg/util/patroni/patroni.go

```diff
@@ -175,6 +177,20 @@ func (p *Patroni) SetConfig(server *v1.Pod, config map[string]interface{}) error
 	return p.httpPostOrPatch(http.MethodPatch, apiURLString+configPath, buf)
 }
 
+// ClusterMembers array of cluster members from Patroni API
+type ClusterMembers struct {
+	Members []ClusterMember `json:"members"`
+}
+
+// ClusterMember cluster member data from Patroni API
+type ClusterMember struct {
+	Name     string `json:"name"`
+	Role     string `json:"role"`
+	State    string `json:"state"`
+	Timeline int    `json:"timeline"`
+	LagInMB  int    `json:"lag"`
```

Comment:
Does Patroni always return lag in MB and not simply in bytes ?
for example, the `patronictl` [converts bytes to MB](https://github.com/zalando/patroni/blob/fce889cd048da605c60fe607f19d7c82f526359d/patroni/ctl.py#L815)

Reply:
renamed to just `Lag`

Reply:
hm maybe lag is reserved for something else:
```
cannot unmarshal string into Go struct field ClusterMember.members.lag of type int
```
renamed it back, since I think it's always in MB.